charts/fybrik/kind-control.values.yaml

```diff
@@ -18,18 +18,21 @@ cluster:
   # Set to the name of the cluster.
   name: kind-control
   # Set to cluster zone.
-  zone: localhost
```

Comment:
in general, why do we need all these yaml files used by  Razee samples and all notebook tests files in the fybrik chart?

Reply:
@roytman as you suggested, the tests related values files are now under test/charts folder